app/bundles/LeadBundle/Entity/LeadListRepository.php

```diff
@@ -855,33 +855,94 @@ public function getListFilterExpr($filters, &$parameters, QueryBuilder $q, $not
 
             // Generate a unique alias
             $alias = $this->generateRandomParameterName();
-
             switch ($details['field']) {
                 case 'hit_url':
-                    $operand = (($func == 'eq') || ($func == 'like')) ? 'EXISTS' : 'NOT EXISTS';
-
+                case 'hit_url_date':
+                case 'referer':
+                case 'source':
+                case 'url_title':
+                case 'lead_email_received_date':
+                case 'notification':
+                    $operand = (($func == 'notEmpty') || ($func == 'between') || ($func == 'eq') || ($func == 'like') || ($func == 'gt') || ($func == 'lt')) ? 'EXISTS' : 'NOT EXISTS';
```

Comment:
I guess, here `in_array` should be used. And also there might be other cases like `lte` an `gte`. So my suggestion is to turn it into
```
$operand = in_array($func, ['notEmpty', 'between', 'eq', 'like', 'gt', 'gte', 'lt', 'lte']) ? 'EXISTS' : 'NOT EXISTS';
```